packages/lesswrong/components/events/EventsHome.tsx

```diff
@@ -0,0 +1,363 @@
+import { Components, registerComponent, } from '../../lib/vulcan-lib';
+import React, { useState, useEffect } from 'react';
+import { useUserLocation } from '../../lib/collections/users/helpers';
+import { useCurrentUser } from '../common/withUser';
+import { createStyles } from '@material-ui/core/styles';
+import * as _ from 'underscore';
+import NotificationsNoneIcon from '@material-ui/icons/NotificationsNone';
+import { useDialog } from '../common/withDialog'
+import {AnalyticsContext} from "../../lib/analyticsEvents";
+import { useUpdate } from '../../lib/crud/withUpdate';
+import { pickBestReverseGeocodingResult } from '../../server/mapsUtils';
+import { useGoogleMaps, geoSuggestStyles } from '../form-components/LocationFormComponent';
+import Button from '@material-ui/core/Button';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import { useMulti } from '../../lib/crud/withMulti';
+import { getBrowserLocalStorage } from '../async/localStorageHandlers';
+import Geosuggest from 'react-geosuggest';
+
+const styles = createStyles((theme: ThemeType): JssStyles => ({
+  section: {
+    maxWidth: 1200,
+    padding: 20,
+    margin: 'auto',
+  },
+  sectionHeadingRow: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    maxWidth: 700,
+    margin: '40px auto',
+    '@media (max-width: 812px)': {
+      flexDirection: 'column',
+      margin: '30px auto',
+    }
+  },
+  sectionHeading: {
```

Comment:
This font is the browser default, but should be styled with serifStack from the theme files. (I don't off the top of my head remember exactly the right way to do this, but it involves spreading one of the theme.typography.something objects.)